onnxruntime/core/providers/xnnpack/nn/conv.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/framework/op_kernel.h"
+#include "core/framework/allocator.h"
+#include "core/providers/cpu/nn/conv_attributes.h"
+#include "core/providers/xnnpack/detail/utils.h"
+#include "xnnpack.h"
+
+namespace onnxruntime {
+class GraphViewer;
+class Node;
+namespace xnnpack {
+
+class Conv : public OpKernel {
+ public:
+  Conv(const OpKernelInfo& info);
+
+  Status Compute(OpKernelContext* /*context*/) const override;
+
+  // use PrePack to handle the weight layout change as that's not a simple NCHW -> NHWC transpose
+  Status PrePack(const Tensor& tensor, int input_idx, AllocatorPtr alloc,
+                 /*out*/ bool& is_packed,
+                 /*out*/ PrePackedWeights* prepacked_weights) override;
+
+  // check to see if an ONNX NCHW Conv node is supported by this implementation. the first input and output will be
+  // converted to NHWC by ORT.
+  static bool IsOnnxNodeSupported(const onnxruntime::Node& nchw_node, const GraphViewer& graph);
```

Comment:
```suggestion
  static bool IsOnnxNodeSupported(const Node& nchw_node, const GraphViewer& graph);
```

Reply:
Needed to disambiguate with OpKernel::Node.